app/src/main/java/com/infomaniak/drive/data/cache/FileController.kt

```diff
@@ -753,6 +754,7 @@ object FileController {
     private fun keepOldLocalFilesData(oldFile: File, newFile: File) {
         newFile.apply {
             children = oldFile.children
+            cursor = oldFile.cursor
```

Comment:
We could improve that function like this maybe?

```kotlin
private fun keepOldLocalFilesData(oldFile: File, newFile: File) = with(newFile) {
        children = oldFile.children
        cursor = oldFile.cursor
        isComplete = oldFile.isComplete
        isOffline = oldFile.isOffline
        responseAt = oldFile.responseAt
        versionCode = oldFile.versionCode
    }
```

Reply:
I agree with the improvement, but in this kind of case it's more of an extension, avoiding the dupplication of `newFile`.